instrumentation/active_job/lib/opentelemetry/instrumentation/active_job/patches/active_job_callbacks.rb

```diff
@@ -21,45 +21,47 @@ def self.prepended(base) # rubocop:disable Metrics/AbcSize, Metrics/MethodLength
                   block.call
                 end
               end
+            end
+          end
 
-              around_perform do |job, block| # rubocop:disable Metrics/BlockLength
-                span_kind = job.class.queue_adapter_name == 'inline' ? :server : :consumer
-                span_name = "#{otel_config[:span_naming] == :job_class ? job.class : job.queue_name} process"
-                span_attributes = job_attributes(job).merge('messaging.operation' => 'process')
+          def perform_now # rubocop:disable Metrics/AbcSize, Metrics/MethodLength, Metrics/CyclomaticComplexity, Metrics/PerceivedComplexity
+            job = self
```

Comment:
@hosamaly Was this just so that we could avoid changing the references to the various attributes below (eg `job.class`, `job.queue_name`)?

If so - do you mind if I just update the references to `self` instead of setting `job = self` ?

Reply:
Indeed, I was trying to avoid modifying lines that didn't need to be modified. But feel free to update it as you see fit.

Reply:
Cool! I will, thank you 😄